frontend-ssr/styles/_variables.scss

```diff
@@ -2,7 +2,13 @@ $colors: (
     white: #fff,
     darker-white: #f8f8f8,
     black: #1b1b15,
-    red: #eb4d4b,
+    green: #58af58,
+    grey: #77838b,
+    silver: #bcc2c6,
+    yellow: #f7df1e,
+    danger: #eb4d4b,
```

Comment:
We decided to call it `red` in this app, so you should have removed `danger` and kept the existing one, to not break other parts of the app.